components/host-dashboard/AddFundsModal.js

```diff
@@ -277,10 +292,16 @@ const AddFundsModal = ({ collective, ...props }) => {
   const { data, loading } = useQuery(addFundsAccountQuery, {
     context: API_V2_CONTEXT,
     variables: { slug: collective.slug },
-    skip: !collective,
   });
   const account = data?.account;
   const currency = account?.currency;
+  let host;
+
+  if (account?.isHost) {
+    host = account;
+  } else {
+    host = account?.host;
+  }
```

Comment:
:ice_cream: Personal preference, a ternary allows us to use a `const` here

```suggestion
  const host = account.isHost ? account : account.host;
```